bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -141,60 +177,87 @@ final class BagsManager(executorService: ExecutorService) {
 
         val key = obs.getNodeKey
         state += key
+        if (bagsStatus(key).isEmpty)
+          setStatus(key, BagsStatus.NewPending.some)
+
         Future {
+          // Wait the delay before running.
+          Thread.sleep(delay)
```

Comment:
This may not be the right place to do it, but I suspect that it should be in the `Future`?
We pass a `delay` to `enqueue` that was never being used to give, in some cases, the system a chance to recover. Do you have any thoughts about this?